web/pandas/pdeps/0014-translate-website-content.md

```diff
@@ -5,137 +5,65 @@
 - Discussion: [#56301](https://github.com/pandas-dev/pandas/issues/56301)
               [#57204](https://github.com/pandas-dev/pandas/pull/57204)
 - Author: [Albert Steppi](https://github.com/steppi),
-- Revision: 1
+- Revision: 2
```

Comment:
the revision number doesn't need to change at this time

(maybe best to not force push changes though to ease review)

Reply:
Thanks. If we continue with the PDEP process, I'll only bump the revision number when asked to. I made the force push to correct a small formatting mistake that I didn't want to pollute git history, but I won't do it again. I'll try to be more careful to fixup mistakes before pushing.